scripts/README.md

```diff
@@ -0,0 +1,17 @@
+# Packaging scripts
+
+This directory contains useful packaging scripts.
+
+## `configure-hypervisor.sh`
+
+This script generates the official set of hypervisor build configuration
```

Comment:
Again, I purposely didn't validate the input, but I can add a check to ensure it is `qemu-lite` if you like. But we should probably permit `qemu` too? Also, we'll need to update it for `qemu-cc` once #34 lands.